### PR TITLE
Algorithm Updated

### PR DESCRIPTION
**Pull request by KongkaMozumderSUST**
#Algorithm resources updated

### CODE REVIEW

**Comment by KongkaMozumderSUST** [comment]
Programming resources

File: books/free-programming-books-bn.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 * [Dynamic Programming Book «ডাইনামিক প্রোগ্রামিং বই»](https://dp-bn.github.io) - Tasmeem Reza, Mamnoon Siam (PDF, [LaTeX](https://github.com/Bruteforceman/dynamic-progamming-book))
 
+* [Number Theory, Graph Theory and LinkedList] (https://progkriya.org/)
```
Comment (sohansai) [inline books/free-programming-books-bn.md:20]: Your contribution is not valid

The title doesn't match the provided link

Read 'docs/CONTRIBUTING.md' before making contributions

**Comment by Ajay-singh1** [comment]
The title is not matching and please remove the trailing slash.

**Comment by eshellman** [comment]
these don't look like books

**Pull request closed without merging**